rfcs/text/0030/user.yml

```diff
@@ -0,0 +1,115 @@
+# Licensed to Elasticsearch B.V. under one or more contributor
+# license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright
+# ownership. Elasticsearch B.V. licenses this file to you under
+# the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# 	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+---
+- name: user
+  title: User
+  group: 2
+  short: Fields to describe the user relevant to the event.
+  description: >
+    The user fields describe information about the user that is relevant
+    to the event.
+
+    Fields can have one entry or multiple entries. If a
+    user has more than one id, provide an array that includes all of
+    them.
+  reusable:
+    top_level: true
+    expected:
+      - client
+      - destination
+      - process
+      - server
+      - source
+      - at: user
```

Comment:
The only changes to user are where it can be nested.

Reply:
For future reference, all we require in this folder is that delta, see https://github.com/elastic/ecs/blob/main/rfcs/text/0008/file.yml. These schema files don't go anywhere, they are purely documentation. So you can keep them as brief as you want

Reply:
Got it. I will remove the stuff already part of these fieldsets.